### ISSUE TITLE
[Doc] seeding database when using Models

### ISSUE THREAD

**Issue by jelhan** [samselikoff/ember-cli-mirage#560]
I'm running 0.2.0-beta.3. I've defined models since I need support for relationships. Seems to work fine. But I haven't any clue how to seed the database. If I'm running `server.create('my-model');` I'm getting an error that I must define a factory. But can't find any information in docs how models and factories play together.

**Comment by jelhan** [comment]
Should have ready [quickstart guide](http://www.ember-cli-mirage.com/docs/v0.2.0-beta.3/quickstart/)... Sorry for the noise.

**Comment by samselikoff** [comment]
Glad you found it. We will be changing this so if you don't have a factory
defined, server.create will fall back to the model.

**Comment by jelhan** [comment]
I was to fast by closing. I thought it's the same as described for [schemas](http://www.ember-cli-mirage.com/docs/v0.2.0-beta.3/quickstart/#associations-and-serializers) but object returned by `server.create('factory-name');` hasn't a create function for relationships.

I thought it's like this:
```
// mirage/models/author.js
import { Model, hasMany } from 'ember-cli-mirage';

export default Model.extend({
  blogPosts: hasMany()
});

// mirage/models/blog-post.js
import { Model } from 'ember-cli-mirage';

export default Model;

// mirage/factory/author.js
port { Factory } from 'ember-cli-mirage';

export default Factory.extend({
  name(i) {
    return `author ${i}`;
  }
});

// mirage/factory/blog-post.js
port { Factory } from 'ember-cli-mirage';

export default Factory.extend({
  title(i) {
    return `title ${i}`;
  }
});

// acceptance test
const author = server.create('author');
autor.createBlogPost();
```

**Comment by samselikoff** [comment]
Yes this is also as an open issue, server.create doesn't currently return
model instances, it only returns attrs. This will change soon.

However, you'll generally want to user server.create, since
author.createBlogPost won't leverage any factory attributes you've defined.

**Comment by jelhan** [comment]
Thanks for your fast response. I still didn't understand how to associate the relationship when using `server.create()`. Could you please give an example?

**Comment by jelhan** [comment]
Is it like this:
```
const blogPost = server.create('blog-post');
const author = server.create('author', {
  blogPosts: [
    blogPost.id
  ]
});
```

And could it be that setting a belongsTo is currently not supported?

```
// mirage/models/blog-post.js
import { Model, belongsTo } from 'ember-cli-mirage';

export default Model.extend({
  author: belongsTo()
});

// acceptance test
const blogPost = server.create('blog-post', {
  author: "1"
});
const author = server.create('author', {
  blogPosts: [
    blogPost.id
  ]
});
```
throws:
> Error: Pretender intercepted GET http://localhost:1337/blogPosts but encountered an error: Mirage: Your handler for the url http://localhost:1337/blogPosts threw an error: newModel.isNew is not a function

**Comment by samselikoff** [comment]
The foreign key is on the child, so do something like

let author = server.create('author')
server.createList('blog-post', {authorId: author.id})

**Comment by samselikoff** [comment]
Closing due to inactivity, let me know if you have any more trouble with the orm.

**Comment by bbtb1982** [comment]
I ran into this error today. The Solution for me was to make sure that I had created the accompanying model and not only a factory. it would be nice to add some error handling that points out that a model is attempting to create a relationship on a model that is no defined. my experience is the errors says something along the lines of cannot find on where() or find() on 'undefined'  which in context makes sense but I think we can add more descriptive errors for this.  let me know if this is a good idea and if so I'll take a stab at adding the error messages.

Super thanks for all work. This add-on is one of my major reasons to why we use ember.
